src/components/calcite-color-picker-hex-input/calcite-color-picker-hex-input.scss

```diff
@@ -2,27 +2,26 @@ $input-side-offset--small: 16px;
 $input-side-offset: 18px;
 
 :host {
-  display: block;
+  @apply block;
 }
 
 .container {
-  width: 100%;
+  @apply w-full inline-grid items-center;
   display: -ms-inline-grid;
-  display: inline-grid;
   -ms-grid-columns: 1fr auto;
   grid-template-columns: 1fr auto;
```

Comment:
Attempt to refactor in terms of the grid utils - or break out separately?

Reply:
Break out, since I have no idea how to do this, and would take me some time to figure out.